nat-lab/tests/test_events_link_state.py

```diff
@@ -576,6 +582,7 @@ async def test_event_link_state_peers_exchanging_data_then_idling_then_resume(
 
 
 @pytest.mark.asyncio
+@pytest.mark.xfail(reason="test flaky - JIRA issue: LLT-4695")
```

Comment:
This one is also not in the latest failures

Reply:
Yep, you're right. Thanks!